anchors.go

```diff
@@ -93,6 +124,14 @@ func (a *Anchors) UnmarshalJSON(data []byte) error {
 	return json.Unmarshal(data, tmp(a))
 }
 
+// UnmarshalJSON is an unmarshaller that handles the variable response from factomd
+func (a *AnchorsFactom) UnmarshalJSON(data []byte) error {
+	type tmp *AnchorsFactom // unmarshal into a new type to prevent infinite loop
+	// json can't unmarshal a bool into a struct, but it can recognize a null pointer
+	data = bytes.Replace(data, []byte("\"factom\":false"), []byte("\"factom\":null"), -1)
```

Comment:
This was a workaround in Anchors.UnmarshalJSON because the Factomd API wrongfully reports it like that and would have required a breaking API change to fix. Since this doesn't have an implemented on the factomd end yet, I see no reason to do it this way and just have the API implementation report a null.